docs/api/device.md

```diff
@@ -416,6 +416,55 @@ if (device.getPlatform() === 'ios') {
 }
 ```
 
+### `device.tap(point, shouldIgnoreStatusBar)`
+
+Perform a tap at arbitrary coordinates on the device's screen.
+
+#### tap parameters
+
+| parameter | platform | description | default value
+|  --------  |  -------  |  -------  | -------  |
+| point | Android & IOS | Coordinates in the element's coordinate space | `{x: 100, y: 100}`
+| shouldIgnoreStatusBar | Android | Coordinates will be measured starting from under the status bar. | `true`
+
+#### tap examples
```

Comment:
This is going to be nitpicky and that's an nice structure, but we usually describe the parameters within bullet points and then just put examples without the headlines. For example: https://wix.github.io/Detox/docs/api/actions#swipedirection-speed-normalizedoffset-normalizedstartingpointx-normalizedstartingpointy

Reply:
fixed